zulip/zulip/__init__.py

```diff
@@ -270,6 +270,8 @@ def get_default_config_filename():
             os.path.exists(os.path.join(os.environ["HOME"], ".humbugrc"))):
         raise ZulipError("The Zulip API configuration file is now ~/.zuliprc; please run:\n\n"
                          "  mv ~/.humbugrc ~/.zuliprc\n")
+    elif not os.path.exists(config_file):
+        config_file = os.path.join(os.environ["PWD"], ".zuliprc")
```

Comment:
This doesn't take into account the case when `.zuliprc` is not present at `$PWD`.

Reply:
That will be checked later. Even in the original code it only raised an error if zuliprc was not present AND humbgrc was present, otherwise it proceeded nornally and was checked later.

Reply:
OK, make sense.